### PR TITLE
Adds COG-Nac, the Justiciar's favorite drink.

### PR DESCRIPTION
**Pull request by UselessTheremin**
## Intent of your Pull Request

Adds a new drink that I was commissioned to do by Alv Cyktor. Sprites and shitcode by me. It is made with Lemon Juice, Cognac, Iron, and Tequila Sunrise. Whenever you drink it, you uncontrollably spin. 

### Why is this good for the game?

More drinks are better, it makes barman a more expansive job if there are more drinks.

#### Changelog

:cl:  
rscadd: Added COG-Nac, a Brass Beverage. 
imageadd: Added a special icon for a glass of COG-Nac
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
how would clock cult get this,  they would have to grow lemons

**Comment by Mqiib** [comment]
image

This is constant spinning. Unsure if this is a good idea.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2249,3 +2249,19 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
 	glass_name = "glass of malt liquor"
 	glass_desc = "A freezing pint of malt liquor."
+
+/datum/reagent/consumable/ethanol/coggernac
+	name = "COG-Nac"
+	description = "I don't even know what an eminence is, but I want him to recall."
+	metabolization_rate = INFINITY
+	boozepwr = 30
+	quality = DRINK_FANTASTIC
+	taste_description = "bronze and brass"
```
Comment (patpol4) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: bronze is fake brass, maybe "cogs and brass"

**Comment by UselessTheremin** [comment]
> image
> 
> This is constant spinning. Unsure if this is a good idea.

pretty sure its just for how long it is in your system, no?

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (UselessTheremin) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: done

**Comment by Mqiib** [comment]
Yes it's as long as it's in your system which could be several minutes of just spinning constantly.

**Comment by UselessTheremin** [comment]
> Yes it's as long as it's in your system which could be several minutes of just spinning constantly.

i tested it in a private server and it was fine so idk, it isnt constant its just frequent just like canned laughter

**Comment by Marmio64** [comment]
should make it so that if you tap the portal with a glass it is filled with COG-Nac, like how using a fork on the portal makes spaghetti

**Comment by SomeguyManperson** [comment]
it'd cause you to spin on reagent cycles so roughly every 2 seconds. Not terribly annoying considering you can spin faster manually though.

**Comment by SomeguyManperson** [comment]
I'd say add a unit of copper in production since that's how you get bronze sheets

**Comment by JamieD1** [comment]
@UselessTheremin Fix conflicts

**Comment by JamieD1** [comment]
Please fix conflicts.

**Comment by UselessTheremin** [comment]
closed due to the person wanting this leaving and my drive to fix it being nonexistant

**Pull request closed without merging**